packages/eslint-plugin/configs/jsdoc.js

```diff
@@ -105,7 +105,15 @@ module.exports = {
 		'jsdoc/require-param-description': 'off',
 		'jsdoc/require-returns': 'off',
 		'jsdoc/require-yields': 'off',
-		'jsdoc/tag-lines': 'off',
+		'jsdoc/tag-lines': [
+			1,
+			'any',
+			{
+				startLines: 0,
```

Comment:
It reports 3401 errors with `startLines` set to `0`. Most of them are incorrect, as we don't want to enforce to have no blank lines after the description in JSDoc:

<img width="1138" alt="Screenshot 2023-05-15 at 10 17 33" src="https://github.com/WordPress/gutenberg/assets/699132/5387c4eb-7d5a-482d-947f-2a6afbb0eed2">

The simplest way would be setting `startLines` to `null`. Then it reports 14 errors fixable with a single `npm run lint:js:fix` call.